### DISCUSSION TITLE
Need a new unique name for REMIX

### DISCUSSION THREAD

**Discussion by putuoka** [remix-run/remix#5674] [Proposals]
It's hard to search on google when using remix so i think it will be great if the name is unique something like remixjs or something so google algorithm know what to search because when i search remix result in dj remix music

**Comment by machour** [comment]
Search "remix.run" on "remix react" :)

**Reply by putuoka** [reply]
> Search "remix.run" on "remix react" :)

ok thanks let me try

**Reply by kiliman** [reply]
Or even better, search for "Michael Jackson Remix" 🤣

**Reply by machour** [reply]
@kiliman I did this in the first days then I facepalmed really hard 😅
